linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/ansible/shared.yml

```diff
@@ -1,14 +1,72 @@
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
 # reboot = false
 # strategy = configure
 # complexity = low
 # disruption = medium
-- name: "Prevent Log In to Accounts With Empty Password - system-auth"
-  replace:
-    dest: /etc/pam.d/system-auth
-    regexp: 'nullok'
+- name: Check if system relies on authselect
+  ansible.builtin.stat:
+    path: /usr/bin/authselect
+  register: result_authselect_present
+
+- name: Check integrity of authselect current profile
```

Comment:
I think this task should have `changed_when: false`, so that it doesn't report as changed for a command that only collects data.

Reply:
Good remind. I will do that. Thanks

Reply:
Done.